addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderThingMonitorHandler.java

```diff
@@ -310,19 +448,28 @@ else if (command == OnOffType.OFF) {
                     break;
             }
         } catch (Exception ex) {
-            logger.error("{}: handleCommand: Command='{}' failed for channel='{}' Exception='{}'", getLogIdentifier(),
-                    command, channelUID.getId(), ex.getMessage());
+            logger.error("{}: handleCommand: Command='{}' failed for channel='{}'", getLogIdentifier(), command,
+                    channelUID.getId(), ex);
         }
     }
 
     @Override
     public void initialize() {
-
         try {
-            super.initialize();
             this.config = getMonitorConfig();
-            logger.info("{}: ZoneMinder Monitor Handler Initialized", getLogIdentifier());
-            logger.debug("{}:    Monitor Id:         {}", getLogIdentifier(), config.getZoneMinderId());
+
+            super.initialize();
+            logger.info("{}: context='initialize' Monitor Handler Initialized", getLogIdentifier());
+
+            dataConverter.addChannel(getChannelUIDFromChannelId(ZoneMinderConstants.CHANNEL_MONITOR_FORCE_ALARM));
+            dataConverter.addChannel(getChannelUIDFromChannelId(ZoneMinderConstants.CHANNEL_MONITOR_EVENT_CAUSE));
+            dataConverter.addChannel(getChannelUIDFromChannelId(ZoneMinderConstants.CHANNEL_MONITOR_RECORD_STATE));
+            dataConverter.addChannel(getChannelUIDFromChannelId(ZoneMinderConstants.CHANNEL_MONITOR_MOTION_EVENT));
+            dataConverter.addChannel(getChannelUIDFromChannelId(ZoneMinderConstants.CHANNEL_MONITOR_DETAILED_STATUS));
+            dataConverter.addChannel(getChannelUIDFromChannelId(ZoneMinderConstants.CHANNEL_MONITOR_ENABLED));
+            dataConverter.addChannel(getChannelUIDFromChannelId(ZoneMinderConstants.CHANNEL_MONITOR_FUNCTION));
+            dataConverter.addChannel(getChannelUIDFromChannelId(ZoneMinderConstants.CHANNEL_MONITOR_EVENT_STATE));
+
         } catch (Exception ex) {
```

Comment:
Please catch specific exceptions instead of catching all exceptions. That way it is more clear what exceptions need to be handled and how this is done.